docs/getting_started.rst

```diff
@@ -22,24 +22,20 @@ See :ref:`optional_installs` for more information.
 
     .. tab-item:: Start locally
 
-        The simplest way to get started is to follow the `getting started 'Start locally' guide for
+        The simplest way to get started is to first follow the `getting started 'Start locally' guide for
         Qiskit <https://qiskit.org/documentation/getting_started.html>`__
 
-        In your virtual environment where you installed Qiskit simply add ``nature`` to the
-        extra list in a similar manner to how the extra ``visualization`` support is installed for
-        Qiskit, i.e:
+        In your virtual environment, where you installed Qiskit, install Qiskit Nature as follows:
 
         .. code:: sh
 
-            pip install qiskit[nature]
+            pip install qiskit-nature
 
-        It is worth pointing out that if you're a zsh user (which is the default shell on newer
-        versions of macOS), you'll need to put ``qiskit[nature]`` in quotes:
-
-        .. code:: sh
-
-            pip install 'qiskit[nature]'
+        .. note::
 
+            As Qiskit Nature depends on Qiskit, you can though simply install it into your
```

Comment:
```suggestion
            As Qiskit Nature depends on Qiskit, you can also simply install it into your
```

I am not sure how to understand the `though` in this sentence. I assume you meant smth along the lines of `also`. If not, feel free to rephrase differently.

Reply:
Again I wanted to update this text but not do too much. For "you can though" you can read it as "you can instead" ie you do not not have to go through the step of separately installing Qiskit.